docs/source/guide/guide-getting-started.rst

```diff
@@ -262,3 +266,63 @@ We can then use this backend for our mitigation.
 
 Note that we don't need to even redefine factories for different stacks. Once
 you have a ``Factory`` it can be used with different front and backends.
+
+Error Mitigation with Probabilistic Error Cancellation
+------------------------------------------------------
+
+In *Mitiq*, it is very easy to switch between different error mitigation methods.
```

Comment:
At some point we really need to decide on a convention for writing Mitiq in text: mitiq or Mitiq or *mitiq* or *Mitiq*? The docs are horribly inconsistent. It probably doesn't matter to anyone but me, but would be nice.

Reply:
FWIW, I vote for Mitiq - capitalized everywhere, no special text.

Reply:
I completely agree! We should converge to a consistent notation in all the docs and README files.
I like the plain and capitalized Mitiq as you suggested.  If I am not wrong this is also what we used in the arXiv paper. 

I unified the notation only in this Getting Started for the moment, since this PR is about this section.

Reply:
Nice! Yes it is "Mitiq" everywhere in the paper. Thanks for this.